setup.cfg

```diff
@@ -2,4 +2,14 @@
 description-file = README.md
 
 [mypy]
+strict = True
+disallow_subclassing_any = False
+
+[mypy-jsonschema.*]
+ignore_missing_imports = True
+
+[mypy-sentry_sdk.*]
+ignore_missing_imports = True
```

Comment:
this should not be required, the Sentry SDK is typed: https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/py.typed